tests/mgbench/benchmark.py

```diff
@@ -201,17 +209,25 @@ def filter_benchmarks(generators, patterns):
     dataset.prepare(cache.cache_directory("datasets", dataset.NAME, dataset.get_variant()))
 
     # Prepare runners and import the dataset.
-    memgraph = runners.Memgraph(
+    vendor = runners.Memgraph(
         args.memgraph_binary,
         args.temporary_directory,
         not args.no_properties_on_edges,
         args.bolt_port,
     )
+    # vendor = runners.Neo4j(
+    #     args.memgraph_binary, 
+    #     args.temporary_directory, 
+    #     args.bolt_port, 
+    #     )
```

Comment:
Put this under if or some other nicer concept, since you have 2 vendors, if seems quite ok from my perspective 😄

Reply:
I updated this and now changed is with arguments.